### PR TITLE
PHP 8.3 gen_stub.php --replace-classsynopses

### PR DESCRIPTION
**Pull request by Girgias**
Created by ``./build/gen_stub.php --replace-classsynopses ./ ./../docs-php/en/`` with some omissions due to false positives in DateTime.

### CODE REVIEW

**Comment by Girgias** [comment]
So GMP doesn't actually document the constructor _yay_

**Review by nielsdos — APPROVED** [review]
OK for ext/dom, thx

File: reference/dom/domelement.xml [hunk 1]
```diff
@@ -47,6 +47,16 @@
      <type>string</type>
      <varname linkend="domelement.props.tagname">tagName</varname>
     </fieldsynopsis>
+    <fieldsynopsis>
+     <modifier>public</modifier>
+     <type>string</type>
+     <varname linkend="domelement.props.classname">className</varname>
```
Comment (kocsismate) [inline reference/dom/domelement.xml:53]: the `domelement.props.classname` description is needed (the same issue below)

**Comment by kocsismate** [comment]
> So GMP doesn't actually document the constructor yay

Yeah, because the constructor was only added recently by Niels :)

File: reference/dom/domelement.xml (hunk 1 above)
Comment (Girgias) [reply reference/dom/domelement.xml:53]: I just ran ``--regenerate-class-synopses``, would be nice if that also would generate the description section :(

File: reference/gmp/gmp.xml [hunk 2]
```diff
@@ -30,6 +30,9 @@
     </ooclass>
 
     <classsynopsisinfo role="comment">&Methods;</classsynopsisinfo>
+    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.gmp')/db:refentry/db:refsect1[@role='description']/descendant::db:constructorsynopsis[@role='GMP'])">
```
Comment (kocsismate) [inline reference/gmp/gmp.xml:33]: The `constructorsynopsis` page should be added in order for it to work.

**Comment by kocsismate** [comment]
Relevant "fix" for the ext/date false positive diff: https://github.com/php/php-src/pull/12566

File: reference/gmp/gmp.xml (hunk 2 above)
Comment (Girgias) [reply reference/gmp/gmp.xml:33]: I know I thought I had merged my GMP changes already

**Pull request merged**